pysnmp/entity/config.py

```diff
@@ -19,6 +20,7 @@
 # Transports
 snmpUDPDomain = udp.snmpUDPDomain
 snmpUDP6Domain = udp6.snmpUDP6Domain
+snmpTCPDomain = tcp.snmpTCPDomain
```

Comment:
Just noting that supporting TCP-over-IP6 would be comprise a great follow-up patch! ;-)